src/Core/src/Graphics/PaintExtensions.cs

```diff
@@ -20,7 +22,11 @@ public static partial class PaintExtensions
 			return null;
 		}
 
-		public static bool IsNullOrEmpty(this Paint? paint)
+		public static bool IsNullOrEmpty(
+#if !NETSTANDARD
+			[NotNullWhen(true)]
```

Comment:
If you merge with main I don't think you'll need this if/def anymore

Reply:
Still is require, because this class is compiled for all the platforms but also in .NET Standard.

Reply:
Peppers added a NotNullWhenAttribute class that's used when you're compiling net7.0 so if you rebase with main this shouldn't be needed